server/chat-commands/moderation.ts

```diff
@@ -126,28 +125,34 @@ exports.commands = {
 		}
 
 		// Only show popup if: user is online and in the room, the room is public, and not a groupchat or a battle.
-		let needsPopup = targetUser && room.users[targetUser.id] && !room.isPrivate && !room.isPersonal && !room.battle;
+		const shouldPopup = (
```

Comment:
Who's decision was this formatting? I'm not sure if I like doing this.

Reply:
I honestly don't think any other format would look better.

Reply:
Feel free to suggest a different one, though?